src/index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Weather project</title>
+        <link rel="stylesheet"
+            href="https://fonts.googleapis.com/css?family=Ubuntu">
+
+        <link rel="stylesheet" type="text/css" href="index_style.css">
+
+    </head>
+    <body>
+        <div class="headerimg">
```

Comment:
For class names lets stick with kebab casing, eg header-img.

For id names we'll prefer snake casing, eg headerImg

Reply:
You can try to use this format for all the class and id names used

Reply:
Sai, we are instructed to use kebab case for both id and class names.